docs/client/concepts/parameter_stores.mdx

```diff
@@ -0,0 +1,30 @@
+---
+title: Parameter Stores
+sidebar_label: Parameter Stores
+slug: /client/concepts/parameter_stores
+---
+
+Parameter Stores are a new way of organizing and managing parameters in your mobile app via the Statsig console.  They are now available for Statsig Android, iOS, and React Native sdks.
+
+## What is a Parameter Store?
+Rather than thinking in terms of Statsig entities (Feature Gates, Experiments, Layers, Dynamic Configs), Parameter Stores allow you to think in terms of *parameters* - things in your app that you want to be configurable remotely.  Parameters decouple your code from the configuration, much like Statsig Layers decouple your code from the specific experiment.  It is a level of indirection that allows you to run any set of experiments, or change gating or values on the fly, without hardcoding an experiment name in your app.  Each of these parameters that you define and check can be remapped at will between statsig entities - provided you maintain the same type for that parameter.  As a result, the parameters will receive different values depending on the `StatsigUser` you initialize the SDK with.  These dynamic values can be updated at any time, without a mobile code change, and without waiting for your mobile release cycle.
+
+## How should I use Parameter Stores?
+Here is a suggested workflow:
+1. Create a parameter store for you team or project.  Parameter Stores are an object to hold related parameters.
+2. As you are adding to your mobile app, consider which variables you'd like to be controlled on the server-side, rather than hard coded into your app. These can be things like:
+   a. A boolean parameter to control access to a new feature.  Even though you may be used to using Feature Gates for boolean based feature management, start with a boolean parameter instead
```

Comment:
This formatting isn't coming through. 

Instead of:
```
2. fooof
  a. something
  b. something else
```

Its coming through as 

```
2. fooof a. something b. something else
```
<img width="716" alt="Screenshot 2024-06-07 at 2 26 36 PM" src="https://github.com/statsig-io/docs/assets/95646168/3e0cc868-4573-4f8b-a4bd-78da9e77933a">